enigma-swing/src/main/java/org/quiltmc/enigma/gui/elements/MenuBar.java

```diff
@@ -590,7 +591,8 @@ private static void prepareThemesMenu(JMenu themesMenu, Gui gui) {
 			themeButton.addActionListener(e -> {
 				UiConfig.setLookAndFeel(lookAndFeel);
 				UiConfig.save();
-				ChangeDialog.show(gui.getFrame());
+				Themes.setupTheme();
+				//ChangeDialog.show(gui.getFrame());
```

Comment:
?

Reply:
oops, was messing around and forgot to revert it seems